backend/app/api/endpoints/admin/device_monitor.py

```
@@ -0,0 +1,282 @@
+                    db, device_kind.user_id, device_id
+                )
+                status_val = device_status.get("status", DeviceStatusEnum.OFFLINE.value)
+                executor_version = device_status.get("executor_version")
+                slot_used = device_status.get("slot_used", 0)
+                slot_max = device_status.get("slot_max", 0)
+            else:
+                status_val = DeviceStatusEnum.OFFLINE.value
+                executor_version = None
+                slot_used = 0
+                slot_max = 0
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)